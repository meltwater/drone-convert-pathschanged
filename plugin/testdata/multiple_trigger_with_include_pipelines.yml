kind: pipeline
name: networkredux/staging

node:
  datacenter: nr-staging

concurrency:
  limit: 1

steps:

- name: inspect
  image: meltwaterfoundation/drone-git
  commands:
  - _scripts/inspect.sh networkredux/staging

- name: verify
  image: meltwaterfoundation/drone-lighter:0.21.0
  commands:
  - _scripts/verify.sh
  when:
    event:
    - pull_request
    - push

- name: deploy
  image: meltwaterfoundation/drone-lighter:0.21.0
  commands:
  - _scripts/deploy.sh networkredux/staging
  when:
    event:
    - push

trigger:
  paths:
    include:
    - networkredux/staging/**
  branch:
  - master

---
kind: pipeline
name: basefarm/production

node:
  datacenter: bf-production

concurrency:
  limit: 1

steps:

- name: inspect
  image: meltwaterfoundation/drone-git
  commands:
  - _scripts/inspect.sh basefarm/production

- name: verify
  image: meltwaterfoundation/drone-lighter:0.21.0
  commands:
  - _scripts/verify.sh
  when:
    event:
    - pull_request
    - push

- name: deploy
  image: meltwaterfoundation/drone-lighter:0.21.0
  commands:
  - _scripts/deploy.sh basefarm/production
  when:
    event:
    - push

trigger:
  paths:
    include:
    - basefarm/production/**
  branch:
  - master

...
