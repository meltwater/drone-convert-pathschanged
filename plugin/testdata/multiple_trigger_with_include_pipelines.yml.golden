kind: pipeline
type: ""
steps:
- commands:
  - _scripts/inspect.sh networkredux/staging
  image: meltwaterfoundation/drone-git
  name: inspect
- when:
    event:
    - pull_request
    - push
  commands:
  - _scripts/verify.sh
  image: meltwaterfoundation/drone-lighter:0.21.0
  name: verify
- when:
    event:
    - push
  commands:
  - _scripts/deploy.sh networkredux/staging
  image: meltwaterfoundation/drone-lighter:0.21.0
  name: deploy
trigger:
  paths:
    include:
    - networkredux/staging/**
  branch:
  - master
  event:
    exclude:
    - '*'
concurrency:
  limit: 1
name: networkredux/staging
node:
  datacenter: nr-staging
---
kind: pipeline
type: ""
steps:
- commands:
  - _scripts/inspect.sh basefarm/production
  image: meltwaterfoundation/drone-git
  name: inspect
- when:
    event:
    - pull_request
    - push
  commands:
  - _scripts/verify.sh
  image: meltwaterfoundation/drone-lighter:0.21.0
  name: verify
- when:
    event:
    - push
  commands:
  - _scripts/deploy.sh basefarm/production
  image: meltwaterfoundation/drone-lighter:0.21.0
  name: deploy
trigger:
  paths:
    include:
    - basefarm/production/**
  branch:
  - master
concurrency:
  limit: 1
name: basefarm/production
node:
  datacenter: bf-production
